2022/incoming/st-nichs-reindeers-need-water.pod

```diff
@@ -0,0 +1,148 @@
+it is during this marathon that they must give it all. And Santa
+makes sure that they have all the hydration and other essentials
+their bodies need. Recently St. Nick's head reindeer trainer has
+purchased new set of I<Camelbak> personal H2O delivery systems
+that is highly tuned for use I<in flight>, which have 3 reservoirs 
+rather than the standard 1, so that they don't get too parched.
+
+Being one who obviously dabbles in Perl on the off season, Santa
+was reminded of a module he recently added to the "Good List". It
+is called L<Util::H2O> and it a relatively new addition to the
+growing list of Perl modules on CPAN that turn C<HASH> references to
+blessed references with automatically generated accessor methods.
+
+Santa has, in fact, been sorely tempted to move this module to
+the C<Very Good Module List> since he's found it so handy with all
+of the Perl programming he does that involves C<HASH> references. He
+finds it particularly useful when wrangling results from L<DBI>'s
+C<selectall_hashref> method, the results produced from his web
+scraping activities using L<Web::Scraper>, or the C<HASH> reference
+he gets after C<decode_json>'ing a block of JSON data from the
+different web API he uses to track the online activities everyone
+on his various I<watch> lists.
+
+Just recently, Santa created this script that uses C<Util::H2O> to
+add accessors to data I<in flight> in order to make it easier to
+manage the response C<HASH> reference returned by L<HTTP::Tiny>.
+
+  use strict;
+  use warnings;
+  use JSON       qw//;
+  use HTTP::Tiny qw//;
+  use Util::H2O; # only exports 'h2o'
+  
+Santa can slip C<h2o> inline to his C<GET> request with more easy than
+he's able to slip into a chimney! Get you spot the addition of C<h2o>?
+
+  # give's Santa "$response->content", "$response->status", "$response->success", etc
+  # from HTTP::Tiny's response object (pure HASH)
+  
+  my $http = HTTP::Tiny->new;
+  my $response = h2o $http->get(q{https://jsonplaceholder.typicode.com/users});
+
+Checking for success is clean and easy,
+
+  # check for unsuccessful web request
+
+  if (not $response->success) {
+  
+    print STDERR qq{Can't get list of online persons to watch!\n};
+    printf STDERR qq{Web request responded with with HTTP status: %d\n}, $response->status;
+    exit 1;
+  
+  }
+  
+As is getting the returned JSON into a C<HASH> reference,
+
+  # decode JSON from response content
+  my $json_array_ref = JSON::decode_json($response->content); # $json is an ARRAY reference
+
+  # iterate over each person, structure
+  # is:
+  # [
+  #  {
+  #    "id": 1,
+  #    "name": "Leanne Graham",
+  #    "username": "Bret",
+  #    "email": "Sincere@april.biz",
+  #    "address": {
+  #      "street": "Kulas Light",
+  #      "suite": "Apt. 556",
+  #      "city": "Gwenborough",
+  #      "zipcode": "92998-3874",
+  #      "geo": {
+  #        "lat": "-37.3159",
+  #        "lng": "81.1496"
+  #      }
+  #    }
+  #  },
+  # ...
+  # ]
```

Comment:
Could we make this a touch clearer? For `is( $got, expected )`, there is a hard-coded `got` and no expected, so it's not immediately clear to me what we're doing here.

Reply:
I am not sure if I removed some `got` or you're referring to something else. I'll make this section more clear, but wanted to see if you could clarify your comments.